app/client/components/user/CreateUserForm.tsx

```diff
@@ -65,7 +65,11 @@ const validate = (values: ICreateUser) => {
 
 class CreateUserForm extends React.Component<InjectedFormProps<ICreateUser, Props> & Props> {
   componentDidMount() {
-    this.props.initialize({ accountType: TransferAccountTypes.USER.toLowerCase(), gender: 'female' });
+    this.props.initialize({
+      accountType: TransferAccountTypes.USER.toLowerCase(),
+      gender: 'female',
+      additionalInitialDisbursement: window.DEFAULT_INITIAL_DISBURSEMENT / 100 || undefined
```

Comment:
I think it would makes sense to deprecate the `starting_balance` config option if we're going down this path. (I do think we should go down this path, it makes sense, though the word "additional" should probably be stripped for the naming convention of the varaibles)

Reply:
oh, didn't even see that as a function.

Reply:
which one should we deprecate?